update/factory/trieSyncersContainerFactory.go

```diff
@@ -121,6 +109,16 @@ func (t *trieSyncersContainerFactory) createOneTrieSyncer(
 	return nil
 }
 
+func trieTopicFromAccountType(accType factory.Type) string {
```

Comment:
suggestion to move this function + factory.Type + factoryTrie.topics in core

Reply:
only used in hardfork.